src/bun.js/node/node_fs.zig

```diff
@@ -5717,8 +5717,16 @@ pub const NodeFS = struct {
         var to_buf: [bun.MAX_PATH_BYTES]u8 = undefined;
 
         if (Environment.isWindows) {
+            const target: [:0]u8 = args.old_path.sliceZWithForceCopy(&this.sync_error_buf, true);
+            // UV does not normalize slashes in symlink targets, but Node does
+            // See https://github.com/oven-sh/bun/issues/8273
+            for (target) |*c| {
+                if (c.* == '/') {
```

Comment:
i feel like there's a SIMD @Vector thing we can do here

but we can leave that as a TODO

Reply:
i'll add a todo

Reply:
sliceWIthZForceCopyWithNormalizedSlashesOnWindows